router/logic_test.go

```diff
@@ -31,7 +31,7 @@ func verifyRoute(t *testing.T, rs *routeSet, inUrl string, expectOutUrl string)
 	}
 }
 
-func TestRouteMatching(t *testing.T) {
+func TestUnitRouteMatching(t *testing.T) {
```

Comment:
I will update the imqs-build.rb file for tests once this is merged in - when i update the submodule on router